Formula/chartmuseum.rb

```diff
@@ -0,0 +1,31 @@
+class Chartmuseum < Formula
+  desc "Host your own Helm Chart Repository"
+  homepage "https://chartmuseum.com"
+  url "https://github.com/helm/chartmuseum/archive/v0.9.0.tar.gz"
+  sha256 "e5fed41b098141a56037340ad3b0857c7defae6d143af56b6003a270cc75631f"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GO111MODULE"] = "on"
+    ENV["GOPROXY"] = "https://gocenter.io"
+
+    system "go",
+        "build",
+        "-v",
+        "-ldflags",
+        "-X main.Version=#{version} -X main.Revision=9382222",
+        "-o",
+        "bin/darwin/amd64/chartmuseum",
+        "cmd/chartmuseum/main.go"
+
+    bin.install "bin/darwin/amd64/chartmuseum"
+  end
+
+  test do
+    assert_match "ChartMuseum version #{version}", shell_output("#{bin}/chartmuseum --version")
```

Comment:
Thanks for the PR, the install looks good on the first try 👍. I just have a comment on the testing.

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?
Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.